packages/magic-link/index.js

```diff
@@ -83,12 +83,18 @@ function MagicLink(options) {
  *
  * @param {object} options
  * @param {string} options.email - The email to send magic link to
+ * @param {string} options.labels - The labels for member
```

Comment:
IMO this module shouldn't necessarily know or care about "labels" - maybe instead it can just receive a payload object like `{payload: {labels: ['label', 'other']}}`

Reply:
Thanks, really helped in cleaning that section up and be more generic. Updated now 😄